AutoRest/Generators/CSharp/CSharp.Tests/Swagger/swagger-mirror-primitives.json

```diff
@@ -185,6 +185,10 @@
                 "string": {
                     "type": "string"
                 },
+                "base64url": {
```

Comment:
It would be nice to have base64url in `path` example in generated code and not just swagger test to make sure the logic for non-body serialization works as well